Dynavity/Dynavity/model/Todo.swift

```diff
@@ -1,10 +1,26 @@
 import SwiftUI
 
-struct Todo: Codable {
-    var id = UUID()
+class Todo {
     var label: String
-    var isCompleted = false
+    var isCompleted: Bool
+
+    init(label: String, isCompleted: Bool) {
+        self.label = label
+        self.isCompleted = isCompleted
+    }
+
+    convenience init(label: String) {
+        self.init(label: label, isCompleted: false)
+    }
 }
 
 // MARK: Hashable
-extension Todo: Hashable {}
+extension Todo: Hashable {
+    static func == (lhs: Todo, rhs: Todo) -> Bool {
+        ObjectIdentifier(lhs) == ObjectIdentifier(rhs)
```

Comment:
```suggestion
        lhs === rhs
```

Would this be better (to avoid the use of `ObjectIdentifier` unnecessarily)? This has exactly the same behaviour: https://github.com/apple/swift/blob/main/stdlib/public/core/Equatable.swift#L253.